startup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# folder where your app is located
+APP_PATH='<APP_PATH>'
+
+### BUILD
+echo 'Building projects...'
+cd $APP_PATH
+cd batch/spark && sbt clean package assembly && cd ../..
+cd streaming/spark && sbt clean package assembly && cd ../..
+cd streaming/flink && sbt clean package assembly && cd ../..
+cd docker
+
+# LOAD CONTAINERS
+echo 'Loading Docker containers...'
+docker network create vnet
+docker-compose -f mongo.yml -f zookeeper.yml -f kafka.yml -f hadoop-hbase.yml -f flink.yml up -d
+docker-compose -f dev/webapp.yml up -d --build
+docker-compose -f dev/batch-spark.yml up -d --build
+docker-compose -f dev/streaming-spark.yml up -d --build
+docker-compose -f dev/streaming-flink.yml up -d --build
+
+# OPEN TERMINALS WITH LOGS
+echo 'Opening Logs...'
+gnome-terminal -- docker logs -f docker_mongo_1 &
```

Comment:
thanks @guilhebl 
compatible only with gnome environment right?

Reply:
@Chabane that's right it might not be portable to all OSs feel free to discard or modify the PR if you think it's not applicable to a broader context

Reply:
you are right

thanks for your contribution 👍